src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -267,6 +269,44 @@ function BulkEmailForm(props) {
         }}
       />
       <Form>
+        {/* this will be pluggable */}
+        <div className="border border-brand-400 p-3">
+          <h1>Pluggable components</h1>
+          <PluggableComponent
+            id="checkbox-form"
+            as="communications-app-check-box-form"
+            label="checkbox label - @openedx-plugins/communications-app-check-box-form"
+            onChange={() => null}
+          >
+            <h1>Checkbox -default</h1>
+          </PluggableComponent>
+
+          <PluggableComponent
+            id="input-form"
+            as="communications-app-input-form"
+            isValid
+            label="Hello"
+            feedbackText="This is okay"
+            controlId="test"
+          >
+            <h1>Input -default</h1>
+          </PluggableComponent>
+          {/* this will return default child if the plugin has not been installed  */}
+          <PluggableComponent id="input-form" as="communications-app-card">
```

Comment:
This points to this plugin: **@openedx-plugins/communications-app-card** but in this case that plugin doesn't exits so this will render the card above